server.js

```diff
@@ -8,6 +8,36 @@ const MongoStore = require('connect-mongo')
 const db = require('./models/db')
 const app = express()
 
+// Start Config //
+// This code makes us have req.body <=====
+app.use(express.urlencoded({ extended: true }))
+app.use((req, res, next) => {
+    res.locals.data = {}
+    next()
+})
+app.use(cors())
```

Comment:
Add `app.use(express.json())`

Reply:
Add above line 18

Reply:
Deleted non needed config and middleware